content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -16,13 +16,64 @@ further_reading:
   text: "Learn more about Security notification variables"
 ---
 
-In addition to the default rules, you can write custom Agent and detection rules. Custom Agent rules are deployed to the Agent in a custom policy separate from the default one. The custom policy contains custom Agent rules as well as [default rules that have been disabled](#disable-default-agent-rules).
+This topic explains how to create custom Datadog Agent and detection rules for [CMS Threats][8].
 
-### Define the agent rule
+In addition to the out of the box (OOTB) [default Agent and detection rules][7], you can write custom Agent and detection rules. Custom rules help to detect events Datadog is not detecting with its OOTB rules.
 
-1. On the [**Agent Configuration**][4] page, click **New Rule**.
+## Custom detection rules summary
+
+Understanding the dependency detection rules have on Agent rules is important when you want to use custom rules. Detection rules are composed of existing, deployed Agent rules and additional expression parameters. 
```

Comment:
```suggestion
Custom detection rules depend on Agent rules. They are composed of existing, deployed Agent rules and additional expression parameters. 
```
I found the copy as written a little hard to parse -- will this work? I wasn't sure I captured your meaning accurately.